lib/todo.rb

```diff
@@ -46,5 +47,9 @@ def remove(item)
       #@TODO: Create remove functionality
     end #remove
 
+    def valid_removal?(response)
+      (1..contents.length+1).member?(response.to_i)
```

Comment:
Apologies, but I typoed here and we do not want +1 because we're not doing a < comparison, which is why one of your tests are failing.
```suggestion
      (1..contents.length).member?(response.to_i)
```